spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -47,11 +49,11 @@ def __call__(self, parser, namespace, values, option_string=None):
         pred = values[::2]
         val = values[1::2]
         if len(pred) != len(val):
-            parser.error("Values for normalization need to be specified for each predictor.")
+            raise parser.error("Values for normalization need to be specified for each predictor.")
```

Comment:
I don't think these `raise` statements are necessary, either? 

The [`.error`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error) method of `ArgumentParser` objects will exit on their own, and don't need `raise` like other error types do.

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/f80dfbc653bb7ae7f4a5d021537b48e43b22f1b7